content/tutorials/history.textile

```diff
@@ -254,22 +284,92 @@ blang[php].
   "See this step in Github":https://github.com/ably/tutorials/commit/868c4c2
 
 blang[swift].
+  We want the retrived history to show in our @ExampleViewController@. In this tutorial we will load retrived messages to a table view. Add @UITableView@ from @Object library@ to your view and bind it in code as "tableView" also remember to change:
+
+  ```[swift]
+    class ExampleViewController: UIViewController {
+  ```
+
+  To:
+
+  ```[swift]
+    class ExampleViewController: UIViewController, UITableViewDataSource {
+  ```
+
+  Our class has to become a delegate for the @dataSource@. In @viewDidLoad()@ add this line:
+
+  ```[swift]
+    tableView.dataSource = self
+  ```
+
+  We will store all the messages retrived from Ably history in array named "historyMessages".
+
+  ```[swift]
+    private var historyMessages: [String] = []
+  ```
+
+  Retrived messages will be shown in table view cells. In @Interface Builder@ choose your @tableView@ in @Attributes Inspector@ find @Prototype Cells@ and change it to 1.
+  <a href="/images/tutorials/tutorials-swift-history-prototype.png" target="_blank">
+    <img src="/images/tutorials/tutorials-swift-history-prototype.png" style="width: 40%" alt="Add prototype cell">
+  </a>
+  Choose your prototype cell in @Attributes Inspector@ find @Style@ and pick "Basic" then find @Identifier@ and change it to "Cell".
+  <a href="/images/tutorials/tutorials-swift-history-prototype-attributes.png" target="_blank">
+    <img src="/images/tutorials/tutorials-swift-history-prototype-attributes.png" style="width: 40%" alt="Change cell style">
+  </a>
+  Navigate to @Size Inspector@ to find @Row Height@ and change it's value to 40.
+  <a href="/images/tutorials/tutorials-swift-history-prototype-height.png" target="_blank">
+    <img src="/images/tutorials/tutorials-swift-history-prototype-height.png" style="width: 40%" alt="Change cell height">
+  </a>
+  Your class has to conformt to @UITableViewDataSource@ protocol. To achive that simply add these three methods to your code:
+
+  ```[swift]
+    func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return historyMessages.count
+    }
+
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) ->
+                    UITableViewCell {
```

Comment:
Well why is UITableViewCell tabbed so far across, shouldn't it be indented on further than `func`?  I tried with my local docs repo and it was the same (the indexPath stuff did not wrap, but the indentation was not right)